v2/bigtable-cdc-to-hbase/src/test/java/com/google/cloud/teleport/v2/templates/BigtableToHbasePipelineIT.java

```diff
@@ -0,0 +1,187 @@
+import com.google.cloud.bigtable.data.v2.models.RowMutation;
+import com.google.cloud.teleport.it.TemplateTestBase;
+import com.google.cloud.teleport.it.bigtable.StaticBigtableResourceManager;
+import com.google.cloud.teleport.metadata.TemplateIntegrationTest;
+import com.google.cloud.teleport.v2.templates.BigtableToHbasePipeline.BigtableToHbasePipelineOptions;
+import com.google.cloud.teleport.v2.templates.utils.HbaseUtils;
+import com.google.cloud.teleport.v2.templates.utils.MutationBuilderUtils;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.beam.sdk.PipelineResult;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.hadoop.hbase.HBaseTestingUtility;
+import org.apache.hadoop.hbase.client.Table;
+import org.apache.hadoop.util.Time;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** End to end table that runs the pipeline from Bigtable to Hbase. */
+@Category(TemplateIntegrationTest.class)
+@TemplateIntegrationTest(BigtableToHbasePipeline.class)
+@RunWith(JUnit4.class)
+public class BigtableToHbasePipelineIT extends TemplateTestBase {
+  private static final Logger LOG = LoggerFactory.getLogger(BigtableToHbasePipelineIT.class);
+
+  // private static StaticBigtableResourceManager bigtableResourceManager;
+  private static BigtableToHbasePipelineOptions pipelineOptions;
+  private static HBaseTestingUtility hBaseTestingUtility;
+  private static Table hbaseTable;
+  private static StaticBigtableResourceManager bigtableResourceManager;
+
+  @BeforeClass
+  public static void setUpCluster() throws Exception {
+
+    // Parse input as though from a live run. This requires passing in params to test via
+    // -Dparameters="..."
+    String input = System.getProperty("parameters");
+    String[] keyValuePairs = input.split(",");
+    Map<String, String> args = new HashMap<>();
+    for (String pair : keyValuePairs) {
+      String[] entry = pair.split("=");
+      args.put(entry[0], entry[1]);
+    }
+
+    // Create pipeline options from some args
+    // Note that we set start and end times in actual test run
+    pipelineOptions = PipelineOptionsFactory.create().as(BigtableToHbasePipelineOptions.class);
+    // Set bigtable change stream options
+    pipelineOptions.setBigtableProjectId(args.get("bigtableProjectId"));
+    pipelineOptions.setInstanceId(args.get("instanceId"));
+    pipelineOptions.setTableId(args.get("tableId"));
+    pipelineOptions.setAppProfileId(args.get("appProfileId"));
+    // Set pipeline options
+    pipelineOptions.setStreaming(true);
+    pipelineOptions.setExperiments(Arrays.asList("use_runner_v2"));
```

Comment:
Have some concerns with requiring `parameters` on the IT. Can we make it assume some defaults? Or maybe a few @Test variants validating different configurations?

Reply:
We expect these parameters to be unnecessary after Cdc GA. The alternative would be to expose our live persistent instances... Not sure what to do here

Reply:
After talking to @Polber , we will remove IT for now until CDC GA and then we can spin up hermetic bigtables with CDC enabled